### PR TITLE
[DGPFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by chivicoko**
- Created travel_sample database
- In the travel_sample database, I created passengers, passengers_details, and accident_cases tables

### CODE REVIEW

**Comment by chivicoko** [comment]
@Mayowa2020 my PR is complaining about logfile. But logfile is there,
and I am sure I have done the test again and again to be certain. It's all checked

File: submissions/victorOkoye/database2/database2.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    PRIMARY KEY (id),
+    FOREIGN KEY (passengers_id) REFERENCES passengers (id)
+);
+
+
+-- creating accident_cases table
+CREATE TABLE accident_cases (
+	id INT NOT NULL AUTO_INCREMENT,
+    passengers_id INT NOT NULL,
+    status INT NOT NULL,
```
Comment (Mayowa2020) [inline submissions/victorOkoye/database2/database2.sql:38]: Change line 38 to:  status  BOOLEAN NOT NULL,

Boolean values are nothing but 0 and 1 i.e. either true or false. if the condition satisfies, it is true else it is false

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @chivicoko! Kindly make all the requested changes and revert when done.

**Comment by chivicoko** [comment]
@Mayowa2020 I have made the corrections you pointed out I should make. The PR still failed.

**Comment by chivicoko** [comment]
still log file issue

**Review by Mayowa2020 — APPROVED** [review]
Well done @chivicoko! **Approved**

**Comment by chivicoko** [comment]
thanks

**Pull request merged**